### PR TITLE
Update bitbucket api

### PR DESCRIPTION
**Pull request by francescogatto**
Updates to v2 api of bitbucket and removing unique for gradle compatibilty

### CODE REVIEW

File: publish-bitbucket.gradle [hunk 1]
```diff
@@ -91,7 +88,7 @@ def urlExists(String repositoryUrl) {
           throw new RuntimeException("Unauthorized BitBucket user - please provide valid username and password in gradle.properties file")
         }
 
-        return (200 == responseCode)
+        return (200 == responseCode || 404 == responseCode)
```
Comment (JeroenMols) [inline publish-bitbucket.gradle]: Why do you treat 404 as a success?

**Review by JeroenMols — APPROVED** [review]
Thanks for contributing this, looks good to me!

How have you tested this?

**Review by JeroenMols — CHANGES_REQUESTED** [review]
Please look into the 404 and answer my question on how you tested this.

File: publish-bitbucket.gradle (hunk 1 above)
Comment (francescogatto) [reply publish-bitbucket.gradle]: my mistake! it can be removed :)

**Comment by JeroenMols** [comment]
Please update the pull request with the change and I'll make sure it gets
merged

**Comment by francescogatto** [comment]
I think I did it! 💃

**Review by JeroenMols — APPROVED** [review]

**Pull request merged**